core/src/feature/container.rs

```diff
@@ -78,27 +77,27 @@ impl<'a> FeatureCollection<'a> for FeatureContainer<'a> {
     }
 }
 
-/// A trait to allow arbitrary types to be potentially created from feature resolution.
+/// A trait to allow arbitrary types to be potentially created from a feature resolution.
```

Comment:
Nitpick here, but I meant "feature resolution" as a general process, there is no "Feature resolution" object so having "a" in front of it is incorrect, I believe. :slightly_smiling_face: